### DISCUSSION TITLE
Get started call to music@iptel.org

### DISCUSSION THREAD

**Discussion by sipsorcery** [fonoster/routr#299] [Q&A] [answered]
I got routr running seamlessly with the docker command below:

`docker run -p 51908:51908 -p 5060:5060/udp -e EXTERNAL_ADDRS=192.168.1.123 fonoster/routr-one:latest`

I also installed the cli tool using:

`npm install --location=global @routr/ctl`

But I'm a bit of a loss at what to do next? If I wanted to place a call from my SIP client to sip:music@iptel.org how would I go about it?

* I have spent an hour or so looking through the docs & other discussions/issues but to no avail.

**Comment by psanders** [comment]
Hey @sipsorcery,

Did iptel.org give you a VoIP account? I'm unfamiliar with that service, but you typically get one.

From the sounds of i,t your network will need: 

1. Create a Peer, in Routr, using iptel.org VoIP account
2. A SIP Domain in Routr
3. An Agent as part of the SIP Domain

Once that is in place, you can register your Agent and call the Peer.

**Reply by sipsorcery** [reply]
Hi @psanders iptel.org is one of the few SIP services that still offer an anonymous end point to call. You don't need a user account on iptel.org (there used to be a lot more services but they've dwindled over the two decades since VoIP/SIP became more mainstream).

Given all that's required is to route the SIP INVITE to iptel.org, even in a stateless SIP Procy mode, does that change what's required for the Routr config? Is there any guide for a very basic SIP Proxy mode of operation?

**Comment by psanders** [comment, marked as answer]
I haven't tried it, but it should work. If anything, you can setup a
placeholder for the credentials (I believe Routr will complain.)

However, since the server is not challenging the request, it should process
it.

If so, please let me know. We should create an issue to remove the
credentials requirement.
--
Pedro

**Reply by sipsorcery** [reply]
Ok thanks.
